amazonka/src/MRIO/Amazonka.hs

```diff
@@ -43,8 +43,7 @@ class HasResourceMap env where
   resourceMap :: env -> ResourceMap
 
 instance HasResourceMap env => MonadResource (RIO env) where
-  liftResourceT (ResourceT run) = do
-    (liftIO . run) =<< asks resourceMap
+  liftResourceT (ResourceT run) = (liftIO . run) =<< asks resourceMap
```

Comment:
@snusnu Good catch. Recent hlint (used via devtools) does not ship this check anymore by default. If you wanted feel free to also PR an updated to `devtools-hs` enabling this flag again.

Reply:
@mbj Aha! I will set some time aside for that later today.